tests/test_date_pyparser.py

```
@@ -3,997 +3,1046 @@
+    # Compressed
+    for case in [('2000001', 2000,   1),
+                 ('1999365', 1999, 365),
+                 ('0000001',    0,   1),
+                 (  '00001', 2000,   1),
+                 (  '50100', 1950, 100)]:
+        test, y, d = case
+        test = before + test + after
+        msg = 'Failure on ' + repr(test)
+        try:
+            pairs = parser_loose.parse_string(test).as_list()
+        except Exception as e:      # pragma: no cover
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)